import-dashboards.py

```diff
@@ -160,10 +160,30 @@ def delete_api_key(db_key, upgrade):
     con.close()
 
 
-def add_datasources(api_key):
-    r = requests.get('%s/api/datasources' % (HOST,), headers=grafana_headers(api_key))
-    print ' * Datasources: %r %r' % (r.status_code, r.content)
-    ds = [x['name'] for x in json.loads(r.content)]
+def _add_metrics_datasource(api_key, ds):
+    if 'Metrics' not in ds:   # https://jira.percona.com/browse/PMM-6518
+        print ' * Adding Metrics Data Source'
+        data = json.dumps({'name': 'Metrics', 'type': 'prometheus', 'jsonData': {'keepCookies': [], 'timeInterval': '1s', 'httpMethod': 'POST'}, 'url': 'http://127.0.0.1:9090/prometheus/', 'access': 'proxy', 'isDefault': True})
```

Comment:
Can two data sources (Metrics and Prometheus below) be the default?

Reply:
Thanks. I have missed it.